operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -366,7 +368,7 @@ type ExposureLoadBalancer struct {
 	//+kubebuilder:validation:Default=false
 	//+kubebuilder:default=false
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1
-	Enabled *bool `json:"enabled,omitempty"`
+	Enabled *bool `json:"isEnabled,omitempty"`
```

Comment:
I think you may not want to change `ExposureLoadBalancer`'s field name because it's something that can break backward compatibility.

Reply:
Oh yes, big thank you for that I had errors in kuttl test validation and was searching what I did wrong with renaming. Here it is.